lib/active_admin_import/dsl.rb

```diff
@@ -35,7 +35,7 @@ module DSL
           flash[:notice] = I18n.t('active_admin_import.imported', count: result.imported_qty, model: model_name, plural_model: plural_model_name)
         end
         if result.has_failed?
-          flash[:error] = I18n.t('active_admin_import.failed', count: result.failed.count, model: model_name, plural_model: plural_model_name)
+          flash[:error] = I18n.t('active_admin_import.failed', count: result.failed.count, model: model_name, plural_model: plural_model_name, message: result.failed_message(limit: 5))
```

Comment:
Line is too long. [184/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.